build.override.targets

```diff
@@ -3,37 +3,39 @@
 
   <PropertyGroup>
     <TraversalBuildDependsOn>
+      WcfCopyProjectJson;
       WcfSetup;
       $(TraversalBuildDependsOn);
       WcfCleanup;
     </TraversalBuildDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <TraversalCleanDependsOn>
       WcfClean;
       $(TraversalCleanDependsOn);
     </TraversalCleanDependsOn>
   </PropertyGroup>
-  
+
   <PropertyGroup>
     <WcfSelfHostSetupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceSetup.log</WcfSelfHostSetupLog>
     <WcfSelfHostCleanupLog>$(MSBuildThisFileDirectory)\SelfHostedWcfServiceCleanup.log</WcfSelfHostCleanupLog>
+    <ScenarioLatestDependencyJson>$(MSBuildThisFileDirectory)\src\System.Private.ServiceModel\tools\test\JsonFiles\stable.dependencies.json</ScenarioLatestDependencyJson>
```

Comment:
Is there some naming mismatch here... the attribute says Scenario**Latest**DependencyJson while the file it points to is **stable**.dependencies.json?